### PR TITLE
Fix lib/easeljs typo getDataUrl

### PR DESCRIPTION
**Pull request by davidholyko**
Fixes typo in lib/easeljs.js where `this.bitmapCache.getDataURL` is undefined

### CODE REVIEW

File: lib/easeljs.js [hunk 1]
```diff
@@ -6696,7 +6696,7 @@ this.createjs = this.createjs||{};
 	 * @return {String} The image data url for the cache.
 	 **/
 	p.getCacheDataURL = function() {
-		return this.bitmapCache?this.bitmapCache.getDataURL():null;
```
Comment (davidholyko) [inline lib/easeljs.js:6699]: this is a typo. the correct function name is getCacheDataURL

https://github.com/CreateJS/EaselJS/blob/master/src/easeljs/filters/BitmapCache.js#L516

**Comment by davidholyko** [comment]
Hi @danzen, I'm not sure exactly what you mean. Can you explain further? Thanks!

**Comment by danzen** [comment]
@davidholyko you found the typo in the DisplayObject.  There is also the same typo in the Bitmap class if you look for bitmapCache.getDataURL().  So would you like to modify your pull request to update both these.  I will then advise that we accept the pull request.  (The only reason I ask is that I have not done a pull request yet - I probably will one day - but it would just be easier if I could advise the crew to accept this one and have it fix both parts - plus, it is your find).  Thanks - if I don't make sense, tell us to handle it from here and we can.  Cheers - and thanks for pointing it out - sorry for the delay.

**Comment by davidholyko** [comment]
@danzen I did not find `bitmapCache.getDataURL()` in [the Bitmap class file](https://github.com/CreateJS/EaselJS/blob/master/src/easeljs/display/Bitmap.js) but I did find `getDataURL` in BitmapCache class

https://github.com/CreateJS/EaselJS/blob/master/src/easeljs/filters/BitmapCache.js#L521

**Comment by davidholyko** [comment]
@danzen Are there any additional steps that I can take to get this merged?

**Comment by danzen** [comment]
Hi @davidholyko - thanks for the reminder.  Just got access to actually merge ;-) and working through setting up the testing environment here.  If you could confirm that the merge is working for you, that would be helpful. Cheers!

**Pull request merged**